### ISSUE TITLE
🚀 Enhancement: Enhance Pro Plan UI by removing un-supported Hourly Backup option

### ISSUE THREAD

**Issue by gurjeetsinghvirdee** [appwrite/appwrite#9036]
### 🔖 Enhancement description

The Pro Plan shows the **Hourly Backup** option, which is meant for the Scale and Enterprise Plan. We would like to remove this option from the Pro Plan UI to avoid confusion and errors and show only the relevant backup options.

### 🎤 Pitch

This change will make things simpler and smoother for users. Currently, the **Hourly Backup** option is available in the Pro Plan, but it’s meant for the Scale or Enterprise Plan. If a Pro Plan user selects it, they’ll get an alert saying `Invalid schedule` or `Invalid parameter for your plan`, which can be confusing and frustrating. By removing this option, we can avoid these errors and ensure users only see the backup options that work for their plan, improving their overall experience.

![Image](https://github.com/user-attachments/assets/f4b8fc28-d246-4f91-be7a-f279b3887f72)

### 👀 Have you spent some time to check if this issue has been raised before?

- [x] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [x] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by DH-555** [comment]
I think it's better leaving it there so people knows that feature actually exists and is available in the scale plan?

**Comment by gurjeetsinghvirdee** [comment]
> I think it's better leaving it there so people knows that feature actually exists and is available in the scale plan?

I agree that leaving the feature visible can help highlight its availability on the Scale plan. To balance this we can leave it there but keep it inactive for **Pro plan** users, clearly indicating that it’s exclusive to the **Scale plan**. This way, users are aware of its existence while understanding it’s not part of their current plan.

**Comment by gurjeetsinghvirdee** [comment]
Hey, @stnguyen90 
Can I work on this issue?

**Comment by HarshMN2345** [comment]
this has been already fixed, closing it 

<img width="1916" height="1162" alt="Image" src="https://github.com/user-attachments/assets/9a15fa45-a46f-457e-96de-8c2d34b073ee" />

<img width="1266" height="878" alt="Image" src="https://github.com/user-attachments/assets/ec8ea993-72ad-432a-9be5-1b0d4f69c348" />
